modules/gitops-manually-aborting-the-rollout.adoc

```diff
@@ -0,0 +1,200 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/getting-started-with-argo-rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-manually-aborting-the-rollout_{context}"]
+= Manually aborting the rollout
+
+When using a canary deployment, the rollout deploys an initial canary version of the application. You can verify it either manually or programmatically. After you verify the canary version and promote it to stable, the new stable version is made available to all users.
+
+However, sometimes bugs, errors, or deployment issues are discovered in the canary version, and you might wish to abort the canary rollout and rollback to a stable version of your application.
+
+Aborting a canary rollout deletes the resources of the new canary version and restores the previous stable version of your application. All network traffic such as ingress, route, or virtual service that was being directed to the canary returns to the original stable version.
+
+The following example procedure deploys a new `red` canary version of your application, and then aborts it before it is fully promoted to stable.
+
+.Procedure 
+
+. Update the container image version and and modify the `.spec.template.spec.containers.image` value from `argoproj/rollouts-demo:yellow` to `argoproj/rollouts-demo:red` by running the following command in the Argo Rollouts CLI:
++
+[source,terminal]
+----
+$ oc argo rollouts set image rollouts-demo rollouts-demo=argoproj/rollouts-demo:red -n <namespace> <1>
+----
+<1> Specify the namespace where the `Rollout` custom resource (CR) is defined.
++
+.Sample output
```

Comment:
```suggestion
.Example output
```
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output 
The guidance doesn't appear to bar "Sample" specifically, but the rest of the repo uses "Example"; so I would guess that someone would search for "Example", not "Sample"